certbot-auto

```diff
@@ -34,7 +34,8 @@ Help for certbot itself cannot be provided until it is installed.
   -n, --non-interactive, --noninteractive   run without asking for user input
   --no-self-upgrade                         do not download updates
   --os-packages-only                        install OS dependencies and exit
-  -v, --verbose                             provide more output
+  -q, --quiet                               provide no output
```

Comment:
Output is still provided when `certbot-auto` upgrades itself. What if we changed the flags description to something like "provide only update/error output" (`certbot` itself will provide error output when given `-q`)?